Frontend/isbit/src/components/label_menu/Menu.vue

```diff
@@ -11,7 +11,7 @@ import ClusterPlot from './ClusterPlot.vue'
       <ClusterPlot />
     </div>
     <div id="text-and-labels">
-      <h2>Uppmärk datapunkt</h2>
+      <h2>Märk Upp Datapunkt</h2>
```

Comment:
Same here, should only be a capital letter in the first word.

Reply:
fixed